config/tasks/restore_state/restore_state.yaml

```diff
@@ -1,2 +1 @@
-# @package _group_
 weight_file: '/home/jtindall/short_tank/short_tank_run_1/outputs/ClassifierBEST.pth'
```

Comment:
this still referring to something from Josh? change to placeholder name instead so user knows to modify

Reply:
Changed this to a placeholder. The only reason to use this config file is if you want to load a state file, and the path to the state file is the only option in the config file, so should hopefully be self-explanatory.